front/components/agent_builder/capabilities/mcp/MCPServerViewsSheet.tsx

```diff
@@ -292,6 +299,15 @@ export function MCPServerViewsSheet({
         );
         if (mcpServerView) {
           setConfigurationMCPServerView(mcpServerView);
+          // If editing a Run Agent or Run Dust App tool, jump to Additional Configuration page
```

Comment:
Why do we need something specific for these servers ? If another server require additional configuration, we're still on the main config screen ?